symbolic-ppdb/src/format/mod.rs

```diff
@@ -295,8 +295,11 @@ impl<'data> PortablePdb<'data> {
     }
 
     /// Reads this file's PDB ID from its #PDB stream.
-    pub fn pdb_id(&self) -> Option<[u8; 20]> {
-        self.pdb_stream.as_ref().map(|stream| stream.id())
+    pub fn pdb_id(&self) -> Option<DebugId> {
+        let raw_id = self.pdb_stream.as_ref().map(|stream| stream.id())?;
+        let (guid, age) = raw_id.split_at(16);
+        let age = u32::from_ne_bytes(age.try_into().unwrap());
+        Some(DebugId::from_guid_age(guid, age).unwrap())
```

Comment:
Do we actually want to panic here? Or maybe 
```suggestion
        DebugId::from_guid_age(guid, age).ok()
```
would be a better choice?

Reply:
I think the `unwrap` is fine. `from_guid_age` only panics if the `guid` part isn't 16 bytes long, and we definitely know this is the case here.

Reply:
I would also prefer `.ok()` as its more concise. As for panics, you are right, the `split_at` above will rather panic when the id buffer is not long enough. Do we have a guarantee for its length?

Reply:
Yes, the `id` field of `PdbStreamHeader` has type `[u8; 20]`. I can change it to `.ok()` anyway though.